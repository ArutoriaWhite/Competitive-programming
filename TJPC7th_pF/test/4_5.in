1238395 1000
1735 1
2139 1
2663 1
2769 1
3450 1
4067 0
4405 1
6640 0
8428 1
9468 0
10067 0
10843 0
11214 0
11613 0
11619 0
16314 1
17783 1
20405 0
22599 1
23307 1
26241 1
27188 0
27318 1
28400 0
29096 1
29154 0
29558 1
32189 0
34788 1
34886 0
35682 0
42978 1
46568 0
46959 0
48080 1
49579 1
50602 0
50943 1
51144 1
51985 1
52427 0
53987 1
54382 0
55123 1
55127 0
55960 0
57231 1
57822 0
58478 1
60112 1
60403 1
60852 1
62228 1
65175 0
65308 1
65675 1
66225 1
68246 0
69982 1
70036 1
70084 0
70610 1
74158 0
74468 1
75370 0
75707 0
76005 0
79061 0
86313 0
89167 0
95489 0
97048 0
99435 0
101177 0
101577 1
101804 0
106358 1
110727 1
111265 1
112374 0
112523 0
113727 0
113945 1
115427 0
115521 0
115554 0
115622 0
118966 0
119541 0
120035 1
123647 0
128001 1
128881 0
129480 1
131356 1
133346 1
133576 0
138546 1
140461 1
142690 0
143999 1
144257 1
147458 1
152014 0
153319 1
153782 1
153984 1
155441 1
155706 1
156821 1
157365 0
158074 1
158430 0
158706 0
159246 1
161183 0
162287 1
163041 0
164792 0
165253 1
166901 1
167895 1
169095 0
169152 0
169295 1
169333 1
172090 0
175467 0
175798 1
176031 0
176748 0
176781 0
176824 1
176970 0
177136 1
178194 0
178385 0
178397 1
179710 0
179856 0
181340 1
181481 0
183169 0
184768 1
185133 1
187631 0
189403 1
189748 0
190454 1
191119 0
191573 0
192825 0
193443 1
194459 0
196820 0
197312 0
197515 1
198963 0
199347 0
199426 0
201602 1
201774 0
202076 1
203535 0
206330 0
206707 1
207082 0
209591 1
209819 0
210565 0
211010 0
212025 0
212265 1
212956 0
213276 1
214398 0
215852 0
217327 0
218696 0
219649 1
220918 1
222308 1
224581 1
225552 1
226535 0
227431 0
230363 0
231006 1
233833 0
236672 0
237415 1
240149 0
240589 1
241640 0
242411 1
242550 1
244680 1
246370 0
247045 1
250243 1
253152 1
254003 0
254573 1
254714 0
255074 1
257340 1
258609 1
259940 0
260002 0
265402 1
266443 1
267307 0
268035 1
268273 0
272609 0
272824 1
272905 0
273161 0
273964 0
274031 0
275137 0
275762 1
276475 1
277774 0
278101 0
278611 1
278746 0
280261 0
281632 1
282977 1
284009 0
287576 1
288951 1
289237 1
289351 1
291982 1
293815 0
294214 1
294617 1
295176 0
295640 0
299078 0
301342 0
302670 1
303802 0
305075 0
306117 1
306196 1
307407 0
309008 0
309773 1
309929 1
311455 0
312348 0
313357 1
313843 1
314982 1
319340 0
319428 1
322470 1
322753 0
323943 0
324227 1
324424 0
325458 1
326612 0
327563 1
328654 0
330993 0
332878 0
333671 1
333936 1
334013 1
334103 1
337109 1
337166 1
338874 1
340160 1
344314 0
344996 1
345233 0
346192 0
347413 0
347651 1
348180 0
348848 1
349180 0
349775 1
349805 0
351881 1
352415 0
353690 0
353765 1
357496 0
362611 1
366502 1
366585 1
366866 1
371154 1
371389 0
373907 0
374008 0
374479 1
375016 0
378720 1
379798 0
379912 1
380308 1
380913 0
381027 0
382423 1
383404 1
384539 0
386159 1
386343 1
386460 1
388354 0
388782 1
391733 0
391790 1
392402 1
392853 0
397632 1
399989 1
400050 0
403288 0
404863 0
404995 1
405652 1
407621 0
409625 1
410360 1
412625 1
414113 1
415167 0
415268 1
415614 0
415829 1
416124 1
421084 0
422096 1
422600 0
423011 1
423864 1
426266 0
426922 0
427258 0
429125 0
429529 0
430730 1
431422 0
431423 1
432539 0
432733 0
433453 1
433524 1
433531 0
433953 1
433968 0
434167 1
435027 1
437547 0
438093 1
440133 0
445460 1
446107 0
449239 0
449463 1
451147 1
452660 0
453411 0
455605 1
458555 1
460769 1
461542 0
465746 0
465931 0
466388 0
466853 1
467026 1
468648 1
468794 1
469034 0
471224 0
471754 0
472232 1
472873 0
475208 1
477945 1
477984 1
480363 1
482071 0
484878 0
485591 0
486300 1
487563 0
487945 1
488581 1
490363 0
490810 0
492333 1
493471 1
494555 0
495193 0
496309 0
498100 1
498265 0
498511 1
499406 1
500232 1
500365 0
502472 1
502709 1
504134 1
506021 0
506248 1
506566 0
506766 0
506885 0
507056 1
508702 0
509005 1
510261 0
511429 0
511596 1
512598 0
514551 1
515554 0
516937 0
519085 0
521857 0
521879 0
525104 0
525854 1
526926 0
528011 0
531355 1
531586 0
532947 0
534045 0
534358 0
535219 1
535364 1
540891 0
544501 0
544591 0
544871 1
546350 0
547379 0
547875 0
548619 1
549491 1
551873 0
552338 1
554055 1
556327 0
557651 0
559264 0
561387 0
562748 1
562931 1
565154 1
565270 0
565516 0
566536 0
567411 1
569121 1
570149 1
570255 0
571550 0
574190 1
574789 1
578042 0
579131 0
579192 0
580662 1
581652 0
582166 0
582856 0
582901 1
585390 0
586053 0
588981 1
589036 1
592744 1
593153 1
593563 0
593681 1
596277 1
599108 0
599262 0
599409 1
599486 0
599714 0
600974 0
601198 1
602426 1
602585 1
602627 0
606985 1
607838 1
609178 0
610059 1
612967 0
613324 1
615443 0
616449 1
616842 0
617043 1
619088 1
619856 1
619880 1
620677 1
621697 1
623158 1
623582 0
624284 1
624951 0
625829 1
626749 0
628695 0
628722 0
632068 1
633521 0
634342 1
634943 0
639086 0
640627 0
641036 0
641215 0
641901 1
642596 1
643888 1
643974 0
644767 0
646134 0
646241 0
646415 1
650930 0
654709 0
656320 1
656651 1
658697 1
659783 0
660541 1
662910 0
663980 0
665236 1
665468 0
666382 0
666444 0
669155 0
670414 0
675346 0
675362 1
676496 0
677120 0
677843 1
678590 0
678644 1
679185 0
679728 0
680352 0
680455 1
680477 1
681307 0
683996 0
684746 0
685768 0
686749 0
686808 0
687671 0
688131 0
689583 0
689694 0
689773 0
690809 0
692323 1
693454 1
693636 1
694490 1
696520 0
696808 0
697780 0
699942 1
700975 1
701319 1
703171 1
703426 1
703549 1
705286 0
707102 0
708588 0
708802 0
709065 0
709920 0
713747 0
713758 1
714709 0
715409 0
715417 0
718294 0
718453 1
719465 0
720247 1
722536 1
722648 0
729884 1
730778 1
730818 1
730880 0
732215 0
732759 1
733628 0
734637 0
735920 1
736797 0
737765 0
739776 1
743020 0
745428 1
747806 1
748584 0
749825 1
750719 0
751971 0
752902 0
753188 1
753542 0
759001 0
760147 0
761189 1
762838 1
764852 0
765767 1
767599 1
768232 1
768533 1
769610 1
770575 1
771205 1
773617 0
774072 0
775383 1
775424 0
776026 0
776488 1
776727 1
777118 0
777178 1
779527 1
782547 0
783035 1
783359 1
787779 0
787845 1
788413 1
790235 0
790859 0
793443 1
794297 0
794642 1
795392 0
796889 1
797240 0
797595 1
798090 1
798878 0
799241 1
800650 0
800759 0
802071 0
803062 0
804258 1
804605 0
806584 1
807070 1
807705 0
810680 1
811491 0
815530 0
815879 1
816046 1
817288 0
818027 0
818377 1
819756 0
821597 1
821800 1
824350 1
824845 1
825193 0
825919 0
826260 1
827802 1
832423 1
833218 0
835687 0
836734 1
836787 1
837736 0
838209 0
840754 0
841798 1
842699 0
847360 1
848055 1
848872 0
851211 0
851646 0
852968 0
854493 0
856220 0
856858 0
857000 0
857679 1
859886 1
861267 1
861810 1
862430 1
863151 0
863965 1
865345 0
866740 0
866906 1
868899 1
875613 1
875648 1
875914 1
875950 1
876620 0
877624 0
878452 1
878518 0
880494 1
880873 0
881038 0
884079 1
885559 0
890803 0
892541 1
893389 1
894975 1
898450 1
899951 0
902605 1
902997 0
903263 0
903787 0
903849 1
904406 0
905059 0
906623 1
908630 1
908766 1
910097 1
910505 0
910563 0
915579 1
916433 0
917325 0
918591 1
918747 0
920151 0
922319 1
922577 0
922759 0
923494 0
925328 0
925356 1
926802 1
927382 1
928758 1
930489 1
936512 1
937491 0
939724 1
941683 1
941772 1
942902 0
943141 1
944547 1
944738 1
945230 1
945302 0
946677 0
948118 1
950414 1
950873 1
951124 1
952753 0
954375 0
957448 0
958039 1
961453 0
962576 0
962802 0
963747 0
965478 0
965869 0
966782 1
967884 0
968137 1
970382 0
970881 0
971214 0
971924 0
972172 1
972354 1
973026 0
974567 1
974959 1
975521 1
975835 1
976467 1
977528 0
979730 1
980115 1
981456 1
981589 0
982842 1
983415 0
985291 0
987502 1
987819 0
987889 0
992425 1
993274 0
994769 1
995153 0
997444 0
998693 0
999392 1
1000841 1
1001523 0
1008057 1
1008903 1
1009612 0
1010896 0
1011495 1
1015224 0
1016574 1
1016733 1
1017241 1
1018021 1
1020232 1
1022770 0
1026380 1
1026752 0
1027112 1
1027474 1
1027659 1
1029765 1
1031961 0
1034708 0
1036340 1
1038255 0
1039608 0
1041167 1
1041342 0
1042805 1
1044584 1
1046767 1
1047797 1
1048585 0
1049165 1
1049784 0
1050847 1
1051956 0
1052619 0
1053358 0
1054491 1
1054807 1
1055169 0
1055215 0
1057148 1
1057410 0
1057805 1
1058334 0
1058829 1
1060356 0
1061460 0
1062553 1
1063120 0
1063159 1
1063390 0
1066549 0
1066681 1
1067658 1
1071497 1
1072142 1
1073669 0
1073737 0
1075407 1
1076114 0
1079284 1
1080541 1
1091732 0
1092012 1
1093888 0
1094557 1
1094706 0
1095039 1
1095481 0
1097457 0
1100111 1
1100671 1
1102323 0
1103699 1
1103758 0
1104292 1
1104616 0
1104767 0
1105467 1
1106717 0
1112544 1
1112759 1
1113161 0
1117816 1
1118214 0
1119359 1
1121555 0
1123050 1
1124546 0
1125205 0
1126521 0
1126655 0
1128561 1
1131370 1
1134038 0
1134415 0
1138034 0
1138861 0
1139711 1
1140189 1
1143415 0
1143941 0
1145268 1
1145840 1
1146131 0
1147537 0
1148586 0
1149453 1
1150407 1
1152084 1
1155326 0
1160359 1
1160818 0
1160908 1
1162801 0
1165984 0
1166019 1
1166353 1
1167085 1
1170014 0
1170083 1
1170571 0
1171395 0
1171808 1
1172662 0
1173441 0
1177816 0
1178931 0
1179048 1
1181786 0
1182212 0
1182791 1
1188852 0
1189286 1
1194332 0
1195519 1
1196130 0
1196466 1
1196769 1
1198703 0
1199913 1
1202131 0
1205010 1
1205417 0
1207401 0
1207502 1
1208659 0
1209778 1
1210329 0
1210887 0
1213495 0
1215227 0
1215447 1
1215842 1
1216210 1
1216270 0
1223468 1
1226094 1
1227653 1
1228682 0
1231157 0
1233381 0
1234946 1
1235045 1
1238323 0
