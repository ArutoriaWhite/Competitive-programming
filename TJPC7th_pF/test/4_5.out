362611 1229617
